toggles/webapp/toggles.js

```diff
@@ -56,5 +56,12 @@ module.exports = {
       description:
         'Displays the latest data additions, that have been added to the API, on the work page.',
     },
+    {
+      id: 'showLocationsAndStatuses',
+      title: 'Show physical locations and statuses of work items',
```

Comment:
Aren't we showing the status of a physical location?

We already show the location.

Sorry to nit pick, but these toggles should be sharable outside of the team and thus clear.

Reply:
what would you propose?

Reply:
Append / hydrate / add the status of an item in the library to physical items.

Reply:
Ah, I see. I am using the toggle to hide both the physical location and the status, though.

Reply:
I'm again being a numpty - grammar wise would it be locations and status?

Reply:
I think they should both be plural – because each work can have multiple items, each of which can have a physical location and status e.g.

1. Closed stores Iconographic: Available
2. Close stores Somewhere else: Unavailable